app/assets/javascripts/base.js.erb

```diff
@@ -245,6 +245,28 @@ var instantClick
     getImageForLink(a);
   }
 
+  // If a link is focused, it is preloaded just like on mousover.
+  // It also covers the issue where a user needs to press <return>
+  // twice in order to follow a focused link.
+  function focusListener(e) {
+    var a = getLinkTarget(e.target)
+
+    if (!a || !isPreloadable(a)) {
+      return
+    }
+
+    a.removeEventListener('focus', focusListener)
```

Comment:
refactor (blocking): There is no need to remove the event listener here because the `focusListener` is on the `document.body`. This line won't fail, but it won't change anything.

```suggestion
```